### ISSUE TITLE
license-list-up-to-date fails because packaging module is missing

### ISSUE THREAD

**Issue by carmenbianca** [fsfe/reuse-tool#497]
Failing run: https://github.com/fsfe/reuse-tool/runs/5064194496?check_suite_focus=true

Error:

```
Traceback (most recent call last):
  File "/home/runner/work/reuse-tool/reuse-tool/.github/workflows/license_list_up_to_date.py", line 16, in <module>
    from packaging import version
ModuleNotFoundError: No module named 'packaging'
Error: Process completed with exit code 1.
```

**Comment by rishawsingh** [comment]
Hii @carmenbianca I would love to work on this issue

**Comment by carmenbianca** [comment]
@rishawsingh By all means :) It's a little bit hard to test, because this job doesn't run on every PR, but have at it.

**Comment by linozen** [comment]
This was likely fixed by #498, don't you think @mxmehl?

**Comment by mxmehl** [comment]
Oh yes, forgot to close this issue, thanks!
